README.md

```diff
@@ -18,11 +18,9 @@ The firmware does not (and likely will not) implement the whole set of AT comman
 
 The major differences are:
 
-~~1. Only the station mode (AT+CWMODE=1) is supported, no AP or mixed mode.~~

```

Comment:
Are you sure all the AT+CWMODEs are supported? I think we support only WIFI_STA (1) and WIFI_AP (2), we don't support the mixed mode WIFI_AP_STA

Reply:
mixed mode is supported

Reply:
AT+CWMODE is mapped directly to WiFi.mode for all 3 values

Reply:
Ok, I misunderstood the code in cmd_AT_CWMODE().